.gitignore

```diff
@@ -1 +1,20 @@
-/nbproject
+# ignore all files
+*
+
+# except our plugins':
+
+!mod/kalvidassign/*
+!mod/kalvidpres/*
+!mod/kalvidres/*
+
+!filter/kaltura/*
+
+!local/kaltura/*
+!local/kalturamediagallery/*
+!local/mymedia/*
+
+!lib/editor/atto/plugins/*
```

Comment:
I think this better be 
!lib/editor/atto/plugins/kalturamedia/*

and same for tinyMCE